pilot/tools/debug/pilot_cli.go

```diff
@@ -211,7 +212,8 @@ func main() {
 	kubeConfig := flag.String("kubeconfig", "~/.kube/config", "path to the kubeconfig file. Default is ~/.kube/config")
 	pilotURL := flag.String("pilot", "localhost:15010", "pilot address")
 	configType := flag.String("type", "lds", "lds, cds, or eds. Default lds.")
-	resources := flag.String("res", "", "Resource(s) to get config for. Should be pod name or app label for lds and cds type. For eds, it is comma separated list of cluster name.")
+	resources := flag.String("res", "istio-ingressgateway-8b76d5856-2qkk7", "Resource(s) to get config for. Should be pod name or app label for lds and cds type. For eds, it is comma separated list of cluster name.")
```

Comment:
We should not hardcode this.

Reply:
will revert change on this file later, still WIP